vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -9,29 +9,28 @@ open FSharp.Compiler.EditorServices
 open FSharp.Compiler.Symbols
 open FSharp.Compiler.Text
 open Hints
-open Microsoft.VisualStudio.FSharp.Editor
 
-module InlineParameterNameHints =
+type InlineParameterNameHints(parseResults: FSharpParseFileResults) =
 
-    let private getParameterHint (range: range, parameter: FSharpParameter) =
+    let getParameterHint (range: range, parameter: FSharpParameter) =
         {
             Kind = HintKind.ParameterNameHint
             Range = range.StartRange
             Parts = [ TaggedText(TextTag.Text, $"{parameter.DisplayName} = ") ]
         }
 
-    let private getFieldHint (range: range, field: FSharpField) =
+    let getFieldHint (range: range, field: FSharpField) =
         {
             Kind = HintKind.ParameterNameHint
             Range = range.StartRange
             Parts = [ TaggedText(TextTag.Text, $"{field.Name} = ") ]
         }
 
-    let private doesParameterNameExist (parameter: FSharpParameter) = parameter.DisplayName <> ""
+    let doesParameterNameExist (parameter: FSharpParameter) = parameter.DisplayName <> ""
 
-    let private doesFieldNameExist (field: FSharpField) = not field.IsNameGenerated
+    let doesFieldNameExist (field: FSharpField) = not field.IsNameGenerated
```

Comment:
Usual naming in many languages for these type of functions is simply "exists", like `List.exists`, `does...Exist` is not very typical. But I'm personally fine with leaving those as they are.

Reply:
Agree, don't know why it's like this, fixed in both locations